tests/phpunit/Unit/PropertyAnnotators/PageContributorsPropertyAnnotatorTest.php

```diff
@@ -98,6 +94,10 @@ public function testAddAnnotation() {
 			->method( 'newUserFromID' )
 			->will( $this->returnValue( $user ) );
 
+		$this->appFactory->expects( $this->once() )
+			->method( 'getUserRights' )
+			->will( $this->returnValue( [] ) );
+
```

Comment:
This does not look good, though that was already the case. See https://www.entropywins.wtf/blog/2016/05/13/5-ways-to-write-better-mocks/